config.sample.yaml

```diff
@@ -67,7 +67,7 @@ services:
     UserID: "@goneb:localhost" # requires a Syncing client
     Config:
       api_key: "qwg4672vsuyfsfe"
-      use_downsized: false
+      image_type: 0
```

Comment:
I'd prefer string enums to integers as strings are more descriptive without knowing the mapping.